src/components/HomePresentation/HomePresentation.tsx

```diff
@@ -2,14 +2,12 @@ import styles from "./HomePresentation.module.css";
 
 const HomePresentation = () => {
   return (
-    <div className={styles.container}>
-      <div style={{ width: "750px", margin: "0 auto" }}>
-        <h1>Travel Time</h1>
-        <span>
-          let's travel with us we will start a new adventure together with a
-          full of happiness and laughs
-        </span>
-      </div>
+    <div className={styles.text_container}>
+      <h1 className={styles.title}>Travel Time</h1>
+      <span className={styles.text}>
```

Comment:
Here it's better to use p instead span, we use span just to help with some inline stylization, a paragraph will be more semantic

```suggestion
      <p className={styles.text}>
```